github_app.go

```diff
@@ -187,6 +188,9 @@ func getInstallationIDFromCache(ctx context.Context, owner, repo string) (int64,
 	installation := &GitHubAppInstallation{}
 	res := GetEnvironment().DB().Collection(GitHubAppCollection).FindOne(ctx, byOwnerRepo(owner, repo))
 	if err := res.Err(); err != nil {
+		if err == mongo.ErrNoDocuments {
```

Comment:
could you reword to errors.Is such as in https://github.com/evergreen-ci/evergreen/blob/13218c35c13cd34e6918ea21d6460bacfec2d6d9/model/host/db.go#L807 ?